src/utils/index.js

```diff
@@ -8,6 +8,7 @@ module.exports.coordinates = require('./coordinates');
 module.exports.checkHeadsetConnected = require('./checkHeadsetConnected');
 module.exports.debug = require('./debug');
 module.exports.entity = require('./entity');
+module.exports.forceCanvasResizeSafariMobile = require('./forceCanvasResizeSafariMobile');
```

Comment:
we can have a namespace is we have more mobile specific utils. I don't think it's worth it for a single one.